src/app/harbor/shop/shop.tsx

```diff
@@ -10,6 +10,7 @@ import { ShopItemComponent } from './shop-item-component.js'
 import { ShopkeeperComponent } from './shopkeeper.js'
 import { safePerson } from '@/app/utils/airtable'
 import Progress from './progress.tsx'
+import { sort } from 'next/dist/build/webpack/loaders/css-loader/src/utils'
```

Comment:
You shouldn't use this import! I think you editor add it automatically.

Reply:
Ah, seems like it. Jetbrains likes to do it